server/config/options.go

```diff
@@ -118,6 +121,9 @@ var DefaultConfig = Config{
 		LogOnly:                  true,
 		EnableNamespaceIsolation: false,
 		AdminNamespaces:          []string{"tigris-admin"},
+		ClientId:                 "",
```

Comment:
Zero values of the type do not need to be assigned explicitly.
Just remove this three lines.
This may be removed as well `EnableNamespaceIsolation: false`

Reply:
removed.